src/main/java/frc/robot/subsystems/shooter/ShooterSS.java

```diff
@@ -0,0 +1,46 @@
+package frc.robot.subsystems.shooter;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.components.TrigonTalonFX;
+import frc.robot.constants.RobotConstants.ShooterConstants;
+import frc.robot.subsystems.TestableSubsystem;
+
+public class ShooterSS extends SubsystemBase implements TestableSubsystem {
+    private final TrigonTalonFX masterMotor;
+
+    public ShooterSS() {
+        masterMotor = ShooterConstants.RIGHT_MOTOR;
+        ShooterConstants.RIGHT_MOTOR.follow(masterMotor);
+        ShooterConstants.LEFT_MOTOR.follow(masterMotor);
+    }
+
+    /**
+     * @param voltage to be set to the motors
+     */
+    public void move(double voltage) {
+        masterMotor.setVoltage(voltage);
+    }
```

Comment:
add set velocity and pidF coefs to local constants

Reply:
What do you mean by "set velocity"?

Reply:
mastermotor.setVelocity().............